book/scripts.md

```diff
@@ -1258,8 +1259,8 @@ reflect the *current* attributes of the element; for example:
 ``` {.javascript .example} 
 element.innerHTML = '<span id=foo>Chris was here</span>';
 element.id = 'bar';
-// Prints "<span id=bar>Chris was here</span>":
 console.log(element.innerHTML);
+// Prints "<span id=bar>Chris was here</span>":
```

Comment:
Not critical IMO

Reply:
Seems just as good to me to put the comment first

Reply:
Yeah I think it’s not critical, but with comment before one might be confused if that’s output from the previous line (like, maybe a print internal to the browser). Confusion would resolve if you think about it but after is unambiguous.